File: testprojects/java/benchmark/domain/language/babbel/LearnArabic.java
package domain.language.babbel;

import infrastructure.asocialmedia.ASocialNetwork;

//Functional requirement 3.1.3
//Test case 91: Class domain.language.babbel.LearnArabic must use infrastructure.socialmedia.SocialNetwork 
//Result: FALSE
public class LearnArabic {
	public LearnArabic(){
		//FR5.1
		new ASocialNetwork();
	}
}

File: testprojects/csharp/benchmark/domain/language/busuu/LearnArabic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.socialmedia;

namespace CSharpBenchmark.domain.language.busuu
{
    //Functional requirement 3.1.3
    //Test case 90: Class domain.language.busuu.LearnArabic must use class infrastructure.socialmedia.SocialNetwork
    //Result: TRUE
    public class LearnArabic
    {
        public LearnArabic()
        {
            //FR5.1
            new SocialNetwork();
        }
    }
}